app/code/core/Mage/ImportExport/Model/Import/Entity/Product/Type/Abstract.php

```diff
@@ -171,7 +171,7 @@ protected function _initAttributes()
                             'is_static'        => $attribute->isStatic(),
                             'apply_to'         => $attribute->getApplyTo(),
                             'type'             => Mage_ImportExport_Model_Import::getAttributeType($attribute),
-                            'default_value'    => strlen($attribute->getDefaultValue())
+                            'default_value'    => strlen($attribute->getDefaultValue() ?? '')
                                                   ? $attribute->getDefaultValue() : null,
```

Comment:
```suggestion
                            'default_value'    => $attribute->getDefaultValue(),
```

Reply:
@kiatng if `$attribute->getDefaultValue()` is an empty string it had returned `null`, now it would be empty string.